src/usr/local/www/firewall_aliases.php

```diff
@@ -145,6 +146,9 @@
 			<td ondblclick="document.location='firewall_aliases_edit.php?id=<?=$i;?>';">
 				<?=htmlspecialchars($alias['name'])?>
 			</td>
+			<td nowrap ondblclick="document.location='firewall_aliases_edit.php?id=<?=$i;?>';">
+				<?=htmlspecialchars($alias_types[$alias['type']])?>
```

Comment:
This should either be wrapped in a check to ensure it exists (e.g. `array_key_exists()`) or use one of the new methods for safely accessing these types of values, e.g. `array_get_path($alias_types, array_get_path($alias, 'type'), gettext("Not Set"))`. In case for some reason a (very old) alias doesn't have a type value in the configuration.